### ISSUE TITLE
How to use MRR ranking metric?

### ISSUE THREAD

**Issue by yaxirhuxxain** [tensorflow/ranking#80]
I have a trained tf.keras based mode. i want to evaluate the model with MRR on test data but i do not get any useful info regarding how i can use TensorFlow ranking API for this purpose. 

I will be grateful if someone can point me to the right direction or a simple example can be of great help. 

Thanks in advance

**Comment by ramakumar1729** [comment]
We currently work with the tf.Estimator API, and don't have a working example with Keras yet.

**Comment by ramakumar1729** [comment]
Closing this for now. Please add any further comments on this to feature request #55.

**Comment by yaxirhuxxain** [comment]
Thanks for your quick response. I have a working example of MRR implemented with tf but having trouble implement it with t.keras.callbacks. If you have time i could share my code with you may be we can implement it as custom metric.
best regards, Yasir 

 

    On Monday, July 15, 2019, 03:52:55 PM GMT+8, Rama Kumar Pasumarthi <notifications@github.com> wrote:  
 
 
We currently work with the tf.Estimator API, and don't have a working example with Keras yet.

**Comment by ramakumar1729** [comment]
Thanks Yasir, that sounds great! I would suggest creating a minimum working example as a pull request, and we can work from there.
